index.js

```diff
@@ -5,12 +5,11 @@ module.exports = appendTransform;
 
 let count = 0;
 
-function appendTransform(transform, ext, extensions) {
+// eslint-disable-next-line node/no-deprecated-api
+function appendTransform(transform, ext = '.js', extensions = require.extensions) {
 	// Generate a unique key for this transform
-	var key = __dirname + count; // eslint-disable-line
+	const key = __dirname + count; // eslint-disable-line no-path-concat
```

Comment:
why not just use `path.resolve` here?

Reply:
Why not.. I'm going to use `path.join` though - same result since `__dirname` is always absolute but I assume one less step / not having to check that it's absolute.